src/App.jsx

```diff
@@ -1,9 +1,13 @@
 import React from "react";
-import Form from "./Form.jsx";
+import Form from "./form/Form";
+import Chat from "./messenger/Chat";
+import "./style.css";
```

Comment:
Why is there CSS here?

Reply:
I expressed myself incorrectly. This style.css file must have global styles.
Styles intended for a component must be at the component level.